kfdef/kfctl_gcp_iap.yaml

```diff
@@ -358,6 +358,16 @@ spec:
         name: manifests
         path: seldon/seldon-core-operator
     name: seldon-core-operator
+  - kustomizeConfig:
+      parameters:
+        - name: user
+          value: SET_EMAIL
```

Comment:
Why do you have dummy parameters here but not in kfctl_basic_auth?

Reply:
kfctl_basic_auth will use default param value "anonymous".

Reply:
Lets add a comment explaining that this gets overwritten by kfctl.